gallery/how_to/work_with_microtvm/micro_train.py

```diff
@@ -71,7 +60,7 @@
 #
 #     .. code-block:: bash
 #
-#       %%bash
+#       %%shell
```

Comment:
What is the difference between `%%bash` and `%%shell`?

Reply:
This is some good info, I wonder if we could capture it somewhere in a comment? (not sure where would be best, so maybe not)

Reply:
I've added a section about this to `docs/README.md`.